lib/driverProviders/sauce.ts

```diff
@@ -63,7 +63,8 @@ export class Sauce extends DriverProvider {
     this.config_.capabilities['username'] = this.config_.sauceUser;
     this.config_.capabilities['accessKey'] = this.config_.sauceKey;
     this.config_.capabilities['build'] = this.config_.sauceBuild;
-    let auth = 'https://' + this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
+    let auth = (this.config_.sauceSeleniumAddressProtocol || 'https://') + this.config_.sauceUser +
```

Comment:
I'd suggest something like

```
let protocol = this.config_.sauceSeleniumUseHttp ? 'http://' : 'https://'
let auth = protocol + this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
```

Also, if you wanted to build the address with the `url` module or template strings or something, that'd also be nifty.